CHANGELOG.md

```diff
@@ -1,21 +1,9 @@
-# NEXT RELEASE
-
-### Enhancements
-* None.
+# 11.3.1 Release notes
 
 ### Fixed
 * Fixed an assertion failure in the sync client when applying an AddColumn instruction for a Mixed column when that column already exists locally. ([#4873](https://github.com/realm/realm-core/issues/4873), since the introduction of Mixed in v11)
```

Comment:
Would be lovely with an example of that assert, so that it's easier to see for users if they could be affected?

Reply:
Can't we be specific about which version is was introduced in?

Reply:
Updated